packages/guides/src/Renderer/TypeRenderer.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace phpDocumentor\Guides\Renderer;
+
+use phpDocumentor\Guides\Handlers\RenderCommand;
+
+interface TypeRenderer
+{
+    public const HTML = 'html';
```

Comment:
I would like to move these contants to their classes, `HtmlTypeRenderer` and `IntersphinxTypeRenderer` That will make this interface really separated from the actual implementation